_includes/head.html

```diff
@@ -1,29 +1,9 @@
 <head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-  <link rel="shortcut icon" href="{{ "/favicon.ico" | prepend:site.url}}">
-  <title>{{ site.title}} - {{ page.title }}</title>
-  <meta name="description" content="{{ site.description }}">
-  <script>
-  (function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)
```

Comment:
It looks like you've removed the google analytics. Please don't do that. I can give you access but we shouldn't drop this.

Reply:
I didn't mean to